src/makeCalculator.js

```diff
@@ -4,7 +4,23 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    operate(action, number) {
+      this.result = action(this.result, number);
```

Comment:
Consider using the arithmetic operators with the assignment operator for better code style. For example, `this.result += number` when the action is addition. Since `operate` is a generic method, you may need to adjust the `add`, `subtract`, `multiply`, `divide` methods to work with the current value of `result` directly.